packages/deployer/subtasks/deploy-contract.js

```diff
@@ -19,15 +18,11 @@ subtask(
     return false;
   }
 
-  const contractArtifacts = await hre.artifacts.readArtifact(contractName);
-  const sourceBytecodeHash = getBytecodeHash(contractArtifacts.deployedBytecode);
-
   // Create contract & start the transaction on the network
   const { contract, transaction } = await _createAndDeployContract(contractName, constructorArgs);
 
   contractData.deployedAddress = contract.address;
   contractData.deployTransaction = transaction.hash;
-  contractData.bytecodeHash = sourceBytecodeHash;
```

Comment:
letting the `initContractData()` do this calculation, we save 1 call to `readArtifacts` fn